lib/matplotlib/table.py

```diff
@@ -629,13 +685,75 @@ def table(ax,
           loc='bottom', bbox=None, edges='closed',
           **kwargs):
     """
-    TABLE(cellText=None, cellColours=None,
-          cellLoc='right', colWidths=None,
-          rowLabels=None, rowColours=None, rowLoc='left',
-          colLabels=None, colColours=None, colLoc='center',
-          loc='bottom', bbox=None, edges='closed')
+    Add a table to an `~.axes.Axes`.
+
+    You have to specify at least one of *cellText* or *cellColours*. These
```

Comment:
Minor preference for passive mode ("At least one of *cellText* or *cellColours* must be specified").